core/src/qt-tray-monitor/authenticate.cc

```
@@ -80,203 +84,125 @@ bool GetAuthenticationResultString(AuthenticationResult err,
-      return AuthenticateWithDirector(jcr, (DirectorResource*)item->resource());
-    case R_CLIENT:
-      return AuthenticateWithFileDaemon(jcr, (ClientResource*)item->resource());
-    case R_STORAGE:
-      return AuthenticateWithStorageDaemon(jcr,
-                                           (StorageResource*)item->resource());
+    case R_DIRECTOR: {
+      auto* sock = jcr->dir_bsock;
+      auto* dir = static_cast<DirectorResource*>(item->resource());
+      hello.bsprintf(Dirhello, monitor->resource_name_, kBareosVersion.Major,
+                     kBareosVersion.Minor, kBareosVersion.Patch);
+
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)